aten/src/ATen/core/DimVector.h

```diff
@@ -5,7 +5,7 @@
 
 namespace at {
 
-constexpr size_t kDimVectorStaticSize = 5;
+constexpr size_t kDimVectorStaticSize = 6;
```

Comment:
This is for get_strides(), right? That function is weird because it stores `collapsed_dims x n_tensors` dimensions whereas DimVector is "normally" just the dims of a single Tensor.

It may just be better to change that function to use `SmallVector<int64_t, 6>` (larger than 6 fine too;  define a new type alias if you like). 

I don't think TensorImpl uses DimVector directly anymore, but it also uses 5 dims. It would be nice to keep this consistent with that.